src/main/resources/application.properties

```diff
@@ -179,3 +179,6 @@ ccd.conditional-apis.case-assigned-users-and-roles.enabled=${CCD_CONDITIONAL_API
 # (Note SecurityLoggingFilter logs at debug)
 security.logging.filter.path.regex=${DATA_STORE_SECURITY_LOGGING_FILTER_PATH_REGEX:.*}
 
+caseworker.role.regex=${CASEWORKER_ROLE_REGEX:caseworker.+}
+caseworker.role.delimiter=${CASEWORKER_ROLE_DELIMITER:-}
+caseworker.role.jurisdiction.element.index=${CASEWORKER_ROLE_JURISDICTION_ELEMENT_INDEX:1}
```

Comment:
I appreciate the intent to make this configurable. 
But I think the rules around the format of the caseworker IDAM role format are very unlikely to change now. I doubt we'll ever need to use a delimiter different that `-` and a jur position different than `1`. Also if that was the case CCD would break in many other places already. We can therefore simplify things a bit removing these 2 config options if we wanted.

Reply:
or, to simplify, couldn't we combine these 3 properties into one property called:
ccd.access-control.caseworker-jurisdiction.regex-group:caseworker-(.+)-.*
?